File: src/main/java/se/kth/spork/exception/ConflictException.java
package se.kth.spork.exception;

/**
 * Thrown when Spork encounters a conflict that cannot be handled properly.
 *
 * @author Simon Larsén
 */
public class ConflictException extends SporkException {
    public ConflictException(String s) {
        super(s);
    }

    public ConflictException(String s, Throwable throwable) {
        super(s, throwable);
    }
}

File: src/main/kotlin/se/kth/spork/exception/ConflictException.kt
package se.kth.spork.exception

/**
 * Thrown when Spork encounters a conflict that cannot be handled properly.
 *
 * @author Simon Larsén
 */
class ConflictException @JvmOverloads constructor(s: String, t: Throwable? = null) : SporkException(s, t)
